src/dbcp/transform/eip_infrastructure.py

```diff
@@ -274,25 +343,34 @@ def projects_transform(raw_proj_df: pd.DataFrame) -> pd.DataFrame:
         col="operating_status",
         val_to_replace="Unknown",
         replacement=pd.NA,
-        expected_count=1,
+        expected_count=2,
     )
-    # pick first of multi-valued entries
+    # # pick first of multi-valued entries
     proj["industry_sector"] = (
         proj.loc[:, "raw_industry_sector"].str.split(",", n=1).str[0]
     ).astype("string")
 
-    duplicative_columns = [  # these are raw names
-        # These columns are just a concatenation of the names and IDs corresponding to the ID columns
-        # They add no information but invite inconsistency
-        "Facility",
-        "Air Construction",
-        "Air Operating",
-        "NGA",
-        "MARAD",
-        "Other Permits",
-    ]
-    duplicative_columns = _format_column_names(duplicative_columns)
-    proj.drop(columns=duplicative_columns, inplace=True)
+    # Coerce lists to strings for SQL
+    proj[
+        ["operating_status_source_documents", "operating_status_source_documents_old"]
+    ] = proj[
+        ["operating_status_source_documents", "operating_status_source_documents_old"]
```

Comment:
I've noticed some columns with the substring "_old" in them. What is the origin of these columns?